fastlane/Fastfile

```diff
@@ -59,6 +59,31 @@ def get_required_env(key)
 end
 
 
+########################################################################
+# Group buildkite logs by action
+########################################################################
+
+# A module that defines methods to be used as overrides to `Fastlane::Actions`
+# module methods.
+module FastlaneActionLogGroup
+  def print_group(action_name)
+    return if %w[is_ci].include?(action_name)
+
+    puts "~~~ 🚖 #{ENV.fetch('FASTLANE_LANE_NAME', '[root]')} >> #{action_name}"
```

Comment:
What do you think of using Buildkite's custom Fastlane emoji here, to make it clearer where the log comes from?

```suggestion
    puts "~~~ :fastlane: #{ENV.fetch('FASTLANE_LANE_NAME', '[root]')} >> #{action_name}"
```